### PR TITLE
remember the password even if it is cleared by the None option

### PR DESCRIPTION
**Pull request by olekstomek**
This improvement of this PR https://github.com/bndw/wifi-card/issues/46. It ensures that the user does not lose the value entered in the password. Currently, if he enters a password and accidentally or out of curiosity clicks `None` and then selects WPA/WEP again, his password will be empty - without a value, he has to re-enter it.
This PR causes the user's password to never be cleared.
Of course, even if this password is remembered, with the `None` option it will not be taken into account when generating the QR Code for the network.

### CODE REVIEW

**Comment by bndw** [comment]
Would it be possible to implement this instead by checking `encryptionMode` at QR Code generation time and if it's `None` using an empty string? If this is possible it would be preferred as it doesn't introduce duplicate state.

Something like,
```js
useEffect(() => {
  ...
  const password = network.encryptionMode === 'None' ? '' : escape(network.password);
  ...
}, ... )
```

**Comment by olekstomek** [comment]
> Would it be possible to implement this instead by checking `encryptionMode` at QR Code generation time and if it's `None` using an empty string? If this is possible it would be preferred as it doesn't introduce duplicate state.
> 
> Something like,
> 
> ```js
> useEffect(() => {
>   ...
>   const password = network.encryptionMode === 'None' ? '' : escape(network.password);
>   ...
> }, ... )
> ```

Thanks for this hint! Done
btw I did a merge via the GitHub web interface. And that's why I lost the formatting and there are more changes to the commit (improved formatting) than I introduced.

File: src/components/Card.js [hunk 1]
```diff
@@ -65,7 +65,8 @@ export const Card = ({ direction = 'ltr' }) => {
     }
 
     const ssid = escape(network.ssid);
-    const password = escape(network.password);
+    const password =
+      network.encryptionMode === 'None' ? '' : escape(network.password);
```
Comment (bndw) [inline src/components/Card.js:69]: Sorry, this was my mistake -- it should be checking for `nopass` instead of `None`

**Review by bndw — COMMENTED** [review]
Aside from the one comment and the conflicts, this looks good to me.

File: src/components/Card.js (hunk 1 above)
Comment (olekstomek) [reply src/components/Card.js]: Thanks, it's done.

**Comment by olekstomek** [comment]
> Aside from the one comment and the conflicts, this looks good to me.

Conflict resolved also.

**Pull request merged**